packages/core/src/extensions/typenames.ts

```diff
@@ -0,0 +1,19 @@
+import { Extension } from '../Extension.js'
+
+export const Typenames = Extension.create({
+  name: 'typenames',
```

Comment:
typenames?

Maybe `namedAttributes`? Or `labeledAttributes`?

Reply:
But this adds the nodetype names of nodes and marks? Wouldn't typenames make sense?

Reply:
typename just is super generic.

It should be clear that this is applying to the HTMLAttributes of nodes & marks, unsure of a good name for that.